ext/background/api.js

```diff
@@ -62,17 +64,17 @@ async function refreshSessions(force) {
         newSession.local_session_expires_at = moment()
           .add(session.session_max_age, 'seconds')
           .toISOString();
-        _sessionCache[slug] = newSession;
+        sessionCache[slug] = newSession;
       } catch (_) {
-        _invalidCookieCache[cookieValue] = slug;
+        invalidCookieCache[cookieValue] = slug;
         console.log(`user is not authenticated with ${slug}`);
       }
     }
   }
 
   chrome.storage.local.get(['active_org'], (resp) => {
```

Comment:
Maybe we can move all the uses of local storage into a shared helper that does error handling and default values?

Reply:
So, I didn't move ALL of the storage API usage... Felt like too deep of a rabbit hole for now. However I did encapsulate a lot of it.